_data/projects/Hoodie.yml

```diff
@@ -1,11 +1,11 @@
 name: Hoodie
-desc: Generic Backend for frontend Apps
+desc: Backend for Offline First Applications
 site: http://hood.ie
 tags:
 - javascript
 - web
 - offlinefirst
 - nobackend
 upforgrabs:
-  name: starter
-  link: http://go.hood.ie/hoodie-starter-issues
+  name: up for grabs
+  link: https://github.com/hoodiehq/camp/issues?utf8=%E2%9C%93&q=label%3A%22up+for+grabs%22
```

Comment:
:lipstick: this can be simplified to `https://github.com/hoodiehq/camp/labels/up%20for%20grabs`